scripts/setup.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# Add yarn to the $PATH so npm cli commands do not fail
+export PATH="${PATH}:$(yarn global bin)"
+
+cd ${OKTA_HOME}/${REPO}
+
+# undo permissions change on scripts/publish.sh
+git checkout -- scripts
+
+# ensure we're in a branch on the correct sha
+git checkout $BRANCH
+git reset --hard $SHA
+
+git config --global user.email "oktauploader@okta.com"
+git config --global user.name "oktauploader-okta"
```

Comment:
not sure if it's necessary from line 9-16. I dont see from SIW repo.
need @kelvinzhu-okta to confirm.

Reply:
One thing is if the permissions change on scripts/publish.sh is problematic, the best way to handle it is actually not to do a git checkout here, but instead to just do a `chmod +x scripts/publish.sh` and then commit that into the repo. That will make the fact that we make whatever script we run executable a no-op.

I also don't see why you need to checkout the sha. Our Bacon infrastructure will already guarantee for you that the currently running sha is checked out for the repo. The only case where this wouldn't happen is if you are triggering this script from another repo, which does not appear to be what you are trying to do here.

In terms of git config, that depends on whether you actually have something that requires that email and username set. We generally actually don't set those values by default, and only do so when merging code to git. I'm not familiar enough with yarn to know whether or not having git configs set is important or not.